worlds/noita/Items.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict
+from BaseClasses import Item
+
+class NoitaItem(Item):
+    game: str = "Noita"
+
+# 110000 - 110013
+item_table: Dict[str, int] = {
+    "Bad":              110000,
+    "Heart":            110001,
+    "Refresh":          110002,
+    "Potion":           110003,
+    "Gold (10)":        110004,
+    "Gold (50)":        110005,
+    "Gold (200)":       110006,
+    "Gold (1000)":      110007,
+    "Wand (Tier 1)":    110008,
+    "Wand (Tier 2)":    110009,
+    "Wand (Tier 3)":    110010,
+    "Wand (Tier 4)":    110011,
+    "Wand (Tier 5)":    110012,
+    "Wand (Tier 6)":    110013,
```

Comment:
Bases on [this](https://github.com/DaftBrit/NoitaArchipelago/blob/9a158b5324e58fecc4349d6a1553b801ddc55d82/init.lua#L50-L55) are these all only wands with "Shuffle: Yes"? Should there also be a set of no shuffle wands?

Reply:
They can be either, those are procedural wands. They have a base class which invokes data\scripts\gun\procedural\gun_procedural.lua which should have weights foir shuffled or unshuffled wands.

Reply:
Ah ok, I only noticed unshuffled variants and thought they were separate.